content/js3/blocks/chaining/index.md

```diff
@@ -0,0 +1,8 @@
++++
+title = 'Chaining Promises'
```

Comment:
Missing content? Remove the block until it's used?

Reply:
FYI Daniel I think there's a bunch of stuff to go on this, but I had to move to other urgent work. Will look when I get a chance